tests/Fixtures/app/console

```diff
@@ -13,8 +13,8 @@ use Symfony\Component\Console\Input\ArgvInput;
 use Symfony\Component\Debug\Debug;
 
 $input = new ArgvInput();
-$env = $input->getParameterOption(array('--env', '-e'), getenv('SYMFONY_ENV') ?: 'dev');
-$debug = getenv('SYMFONY_DEBUG') !== '0' && !$input->hasParameterOption(array('--no-debug', '')) && $env !== 'prod';
+$env = $input->getParameterOption(array('--env', '-e'), $_SERVER['APP_ENV'] ?? null ?: 'test');
```

Comment:
`$_SERVER['SYMFONY_DEBUG'] ?? 'test'`?

Reply:
Didn't get that? This flag is to pilot the debug boolean, env is 'test' by default if no app_env is defined?